src/components/Confirm/style.css

```diff
@@ -18,9 +18,16 @@
   padding: 0;
 }
 
-.fileViewer{
-  width: 92%;
-  height: 290px;
-  max-height: 70vh;
-  border: 0;
+:global {
+  #pdfIcon {
```

Comment:
Yeah, I know, I was being stupid, I was passing a string when I can just use the embedded html with css modules syntax